api/src/main/java/org/openmrs/api/AdministrationService.java

```diff
@@ -293,8 +293,6 @@
 	 * (presentation layer). This set includes all the available locales (as indicated by the
 	 * MessageSourceService) filtered by the allowed locales (as indicated by this
 	 * AdministrationService).
-	 * 
-	 * @return list of allowed presentation locales TODO change this return type to list?
```

Comment:
We still need the return javadoc. The only test to be removed is: "TODO change this return type to list?"